radio/src/gui/480x272/model_select.cpp

```diff
@@ -239,36 +239,58 @@ void onDeleteModelConfirm(const char * result)
   }
 }
 
+bool isModelOff() {
+  if (TELEMETRY_STREAMING()) {
+    RAISE_ALERT(STR_MODEL, STR_MODEL_STILL_POWERED, STR_PRESS_ENTER_TO_CONFIRM, AU_MODEL_STILL_POWERED);
+    while (TELEMETRY_STREAMING()) {
+      RTOS_WAIT_MS(20);
+      if (readKeys() == (1 << KEY_ENTER)) {
+        killEvents(KEY_ENTER);
+        return true;
+      }
+      else if (readKeys() == (1 << KEY_EXIT)) {
+        killEvents(KEY_EXIT);
+        return false;
+      }
+    }
+  }
+  return true;
+}
+
 void onModelSelectMenu(const char * result)
 {
   if (result == STR_SELECT_MODEL) {
-    // we store the latest changes if any
-    storageFlushCurrentModel();
-    storageCheck(true);
-    memcpy(g_eeGeneral.currModelFilename, currentModel->modelFilename, LEN_MODEL_FILENAME);
-    modelslist.setCurrentModel(currentModel);
-    modelslist.setCurrentCategory(currentCategory);
-    loadModel(g_eeGeneral.currModelFilename, true);
-    storageDirty(EE_GENERAL);
-    storageCheck(true);
-    chainMenu(menuMainView);
+    if (isModelOff() && !g_eeGeneral.disableRssiPoweroffAlarm) {
```

Comment:
Shouln't those checks be swapped so that alert is not raised if not desired ?

Reply:
Indeed the this wasn't working when rssi check was disabled...Thanks. Fixed